<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiEntropyAnalyzer</class>
 <widget class="QMainWindow" name="UiEntropyAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entropy Analyzer Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_AnalyzeInput">
      <property name="text">
       <string>Analyze input</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_InputBytes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton_LoadFiles">
      <property name="text">
       <string>Load files</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="AdvancedPlainTextEdit" name="plainTextEdit_Output"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox_BinaryMode">
      <property name="text">
       <string>Bnary mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>extensions/AdvancedPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
